king_phisher/client/assistants/campaign.py

```diff
@@ -262,6 +439,23 @@ def _get_graphql_company(self, company_name):
 		}""", {'name': company_name})
 		return results['db']['company']
 
+	def signal_url_treeview_row_activated(self, treeview, path, column):
+		model_row = self._url_model[path]
+		self._set_info(model_row)
+
+	def _set_info(self, model_row):
```

Comment:
This should be a bit more descriptive since we have other `_set_info` methods. Rename it to `_set_info_url_details`